move-the-ball.js

```diff
@@ -0,0 +1,32 @@
+const field = document.getElementById('field');
+const ball = document.getElementById('ball');
+
+field.addEventListener('click', (event)=> {
+  const innerFieldCoords = {
+    y: event.clientY - (field.offsetTop + field.clientTop),
+   x: event.clientX - (field.offsetLeft + field.clientLeft),
+  }
+  console.log(innerFieldCoords);
+  const ballCoords = {
+    x: innerFieldCoords.x - ball.offsetWidth/2,
+    y: innerFieldCoords.y - ball.offsetHeight/2
+  }
+  console.log(ballCoords);
+  if (ballCoords.y < 0) {
+   ballCoords.y = 0;
```

Comment:
it is a bad practice to change object properties directly, use multiple variables or create a class with methods for change